### PR TITLE
[VEN-2241]: deployments of XVS bridge on supported testnet networks

### PR DESCRIPTION
**Pull request by GitGuru7**
Deployment of the code included here: https://github.com/VenusProtocol/token-bridge/pull/12

## Checklist

- [ ] I have updated the documentation to account for the changes in the code.
- [ ] If I added new functionality, I added tests covering it.
- [ ] If I fixed a bug, I added a test preventing this bug from silently reappearing again.
- [ ] My contribution follows [Venus contribution guidelines](docs/CONTRIBUTING.md).

### CODE REVIEW

**Review by chechu — DISMISSED** [review]

File: deployments/bsctestnet/DefaultProxyAdmin.json
Comment (chechu) [inline deployments/bsctestnet/DefaultProxyAdmin.json:1]: We should reuse the DefaultProxyAdmin deployed in the Oracle project: 0xef480a5654b231ff7d80A0681F938f3Db71a6Ca6

https://github.com/VenusProtocol/oracle/blob/develop/deployments/bsctestnet/DefaultProxyAdmin.json

To do that, we can copy that deployment file to the deployments folder of this project and redeploy the contracts. hardhat should reuse the already deployed contract

File: deployments/bsctestnet_addresses.json [hunk 1]
```diff
@@ -2,10 +2,10 @@
   "name": "bsctestnet",
   "chainId": "97",
   "addresses": {
-    "DefaultProxyAdmin": "0x43fd9F2DAcB36853dBfBa269A7539706e787cFce",
-    "XVSBridgeAdmin": "0x5D08D49A2e43aC4c72C60754d1550BA12e846d66",
-    "XVSBridgeAdmin_Implementation": "0x8a4C5090d972C83C44E8132a1Fc2Ff3E948E9A71",
-    "XVSBridgeAdmin_Proxy": "0x5D08D49A2e43aC4c72C60754d1550BA12e846d66",
-    "XVSProxyOFTSrc": "0x963cAbDC5bb51C1479ec94Df44DE2EC1a49439E3"
+    "DefaultProxyAdmin": "0x7877ffd62649b6a1557b55d4c20fcbab17344c91",
```
Comment (chechu) [inline deployments/bsctestnet_addresses.json:5]:
```suggestion
    "DefaultProxyAdmin": "0xef480a5654b231ff7d80A0681F938f3Db71a6Ca6",
```

File: deployments/opbnbtestnet_addresses.json [hunk 2]
```diff
@@ -2,6 +2,11 @@
   "name": "opbnbtestnet",
   "chainId": "5611",
   "addresses": {
-    "XVS": "0x3d0e20D4caD958bc848B045e1da19Fe378f86f03"
+    "DefaultProxyAdmin": "0xB1281ADC816fba7df64B798D7A0BC4bd2a6d42f4",
```
Comment (chechu) [inline deployments/opbnbtestnet_addresses.json:5]: Verify this contract in opbnbscan

File: deployments/opbnbtestnet_addresses.json [hunk 3]
```diff
@@ -2,6 +2,11 @@
...
+    "XVS": "0xc2931B1fEa69b6D6dA65a50363A8D75d285e4da9",
+    "XVSBridgeAdmin": "0x19252AFD0B2F539C400aEab7d460CBFbf74c17ff",
```
Comment (chechu) [inline deployments/opbnbtestnet_addresses.json:7]: Verify this contract in opbnbscan

File: deployments/sepolia_addresses.json [hunk 4]
```diff
@@ -2,11 +2,11 @@
   "name": "sepolia",
   "chainId": "11155111",
   "addresses": {
-    "DefaultProxyAdmin": "0xf707294B965B57ca16f191D8015AAB72627C7d6e",
-    "XVS": "0xDb633C11D3F9E6B8D17aC2c972C9e3B05DA59bF9",
-    "XVSBridgeAdmin": "0x4A4a5f6Ecc4DB4ad199E8f1Db388FFFb707fad52",
-    "XVSBridgeAdmin_Implementation": "0x9a59dCAB13883D6717098816210D3dF4B271340B",
-    "XVSBridgeAdmin_Proxy": "0x4A4a5f6Ecc4DB4ad199E8f1Db388FFFb707fad52",
-    "XVSProxyOFTDest": "0x307C77D8606d7E486aC5D73d309e16996A336dbd"
+    "DefaultProxyAdmin": "0xe98a3110929c6650c73031756288Ec518f65e846",
```
Comment (chechu) [inline deployments/sepolia_addresses.json:5]:
```suggestion
    "DefaultProxyAdmin": "0xd3c6bdeeadB2359F726aD4cF42EAa8B7102DAd9B",
```

File: deployments/sepolia/DefaultProxyAdmin.json
Comment (chechu) [inline deployments/sepolia/DefaultProxyAdmin.json:1]: I think here it's happening the same thing happening for the bsctestnet deployment: the DefaultProxyAdmin actually used is the Oracle's one (`0x01435866babd91311b1355cf3af488cca36db68e`, that is ok), but in the json file we have the data of the IL deployment.

I would copy the DefaultProxyAdmin.json from the oracle repo: https://github.com/VenusProtocol/oracle/blob/develop/deployments/sepolia/DefaultProxyAdmin.json

File: helpers/deploymentConfig.ts [hunk 5]
```diff
@@ -57,18 +59,38 @@ export const xvsBridgeMethods = [
...
+  "setPayloadSizeLimit(uint16,uint256)",
+  "setWhitelist(address,bool)",
+  "setConfig(uint16,uint16,uint256,bytes)",
+  "sweepToken(address,address,uint256)",
+  "updateSendAndCallEnabled(bool)",
 ];
 
+export const XVSBridgeAdminMethods = ["setTrustedRemoteAddress(uint16,bytes)", "transferBridgeOwnership(address)"];
+
+export const XVSTokenDestMethods = ["migrateMinterTokens(address,address)", "setMintCap(address,uint256)"];
```
Comment (chechu) [inline helpers/deploymentConfig.ts:86]: I think we should add these entries:
* updateBlacklist(address,bool)
* pause()
* unpause()

File: deployments/opbnbtestnet_addresses.json (hunk 2 above)
Comment (GitGuru7) [reply deployments/opbnbtestnet_addresses.json:5]: `DefaultProxyAdmin` Verified

**Pull request merged**